plugin/evm/validators_normalizer_test.go

```diff
@@ -23,18 +23,14 @@ func (t TestValidatorsNormalizer) ByEpoch(e uint64) (map[ids.ShortID]uint64, err
 func TestValidatorsNormalizer_ByEpoch(t *testing.T) {
 	t.Run("nominal case", func(t *testing.T) {
 		testValidators := validatorsData
-		testWeightRatios := map[ids.ShortID]uint64{
-			{1}: 13,
-			{2}: 37,
-		}
+		testWeightRatios := validatorsData
```

Comment:
I didn't fully understand what was wrong in this case before. Can you explain?

Reply:
This test uses a hardcoded initial value:
```
validatorsData = validatorsTestData{
		uint64(0): map[ids.ShortID]uint64{
			ids.ShortID([20]byte{202, 22, 105, 80, 240, 156, 96, 235, 127, 79, 82, 60, 159, 12, 172, 216, 171, 131, 235, 63}):   6185718675989118760,
			ids.ShortID([20]byte{182, 115, 39, 234, 78, 181, 63, 96, 108, 42, 134, 160, 148, 157, 103, 198, 77, 146, 118, 200}): 6465383072520688749,
			ids.ShortID([20]byte{20, 206, 223, 250, 40, 74, 158, 159, 174, 134, 44, 20, 207, 134, 220, 231, 63, 247, 84, 240}):  515007957320493220,
		},
		uint64(1): map[ids.ShortID]uint64{
			ids.ShortID([20]byte{231, 37, 78, 169, 39, 173, 150, 37, 20, 7, 91, 212, 235, 6, 235, 74, 120, 22, 233, 150}):     9599604527249266409,
			ids.ShortID([20]byte{236, 230, 116, 165, 56, 37, 35, 211, 216, 60, 90, 243, 59, 36, 173, 140, 109, 252, 169, 32}): 3417212044008417831,
		},
	}```

Each of the keys have zero weight after ratio calculation inside `calcWeightRatio`.
in order to be compared for testing, the map that it is compared against need to have same keys with zero values, not the previously existing map
```
testWeightRatios := map[ids.ShortID]uint64{
			{1}: 13,
			{2}: 37,
		}
```